seeds/reviewSeeds.json

```diff
@@ -0,0 +1,92 @@
+[
+ {
+          "rating": 2,
+          "review":"Dwight was on awesome until I felt he got comfortable in our sessions. In one session we had, Dwight could tell that I had not studied the night prior. He told me if I did not have time to study then he did not have time to waste on me that could be better spent with M on this years beet harvest.",
```

Comment:
Our review model is currently set at 100 characters, so it'll error out on this one, I think.

Maybe we could change the limit in /models/Review.js to accommodate longer reviews?